app/controllers/tournaments_controller.rb

```diff
@@ -0,0 +1,40 @@
+class TournamentsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def saveTournamentResults
+    #byebug
+    
+    #TODO: adding validation to the models
+    tournament = Tournament.new
+
+    tournament.tournament_id = params["tournamentId"]
+    tournament.start_date = params[:startDate]
+    tournament.end_date = params[:endDateTime]
+    results = params[:results]
+    results.each do |i|
```

Comment:
Better to use a meaningful param name
```suggestion
    results.each do |result|
```